.github/workflows/lint_python.yml

```diff
@@ -6,16 +6,31 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v3
-      - run: pip install --upgrade pip wheel
-      - run: pip install black flake8 flake8-2020 flake8-bugbear
-                         flake8-comprehensions mypy pytest pyupgrade safety
-      - run: black --check . || true
-      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-      - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88
-                      --show-source --statistics
-      - run: pip install -r pex-requirements.txt -r tests/requirements.txt
-      - run: mkdir --parents --verbose .mypy_cache
-      - run: mypy --ignore-missing-imports --install-types --non-interactive . || true
-      - run: pytest . || true
-      - run: pytest --doctest-modules . || true
-      - run: safety check
+        with:
+          cache: 'pip'
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip wheel
+          pip install .[all]
+      - name: Lint with black
+        run: black --check . || true
+      - name: Lint with flake8
+        run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - name: Lint with flake8 strictly
+        run: |
+          flake8 . \
```

Comment:
I do not believe that yaml rules dictate that the \ is required in this context.